portal/src/main/webapp/js/src/patient-view/util/ClinicalAttributesUtil.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+/**
+ * Functions for dealing with clinical attributes.
+ */
+window.ClinicalAttributesUtil = (function() {
+  /**
+   * Clean clinical attributes. Useful for rounding numbers, or other types of
+   * data cleanup steps. Probably differs per institution.
+   * @param {object} clinicalData - key/value pairs of clinical data
+   */
+  function clean(clinicalData) {
+    // Shallow Copy clinicalData
+    var cleanClinicalData = $.extend({}, clinicalData);
+    var NULL_VALUES = [
```

Comment:
@pieterlukasse @fedde-s Sorry I actually didn't see these comments until today.

I made some comments further up regarding the config. If you have an immediate need for changing the styling of the patient view page, we can facilitate something like that through changing the properties file as proposed. Otherwise I feel hesitant to introduce new config until we have better defined the current config procedure. I'd like to see consensus on these first https://github.com/cBioPortal/cbioportal/pull/1517 and https://github.com/cBioPortal/cbioportal/pull/1516 .